src/test/java/org/cactoos/io/TeeInputStreamTest.java

```diff
@@ -67,4 +69,39 @@ public void copiesContentByteByByte() throws IOException {
         );
     }
 
+    @Test
+    public void leftInputUnclosed() {
+        try (StringWriterMock write = new StringWriterMock()) {
+            new LengthOf(
+                new TeeInput(
+                    "foo",
+                    new OutputTo(write)
+                )
+            ).intValue();
+            MatcherAssert.assertThat(
+                "Can''t use output after usage from TeeInput",
+                write.isClosed(),
+                Matchers.equalTo(false)
+            );
+        }
+    }
+
+    /**
+     * Mock object around StringWriter for checking closing state.
+     */
+    private static final class StringWriterMock extends StringWriter {
+        /**
+         * Closing state.
+         */
+        private final AtomicBoolean closed = new AtomicBoolean(false);
+
+        @Override
+        public void close() {
+            this.closed.set(true);
+        }
+
+        public boolean isClosed() {
```

Comment:
@SharplEr I don't remember exactly where I saw that, but there was something about right method naming rules: just `closed()` should be enough in this case. Sorry, but I can't share a link (maybe it was in EO book).

Reply:
@proshin-roman Name `closed()` violates the code style check `AvoidFieldNameMatchingMethodName`. For example, class `EmptyClosableReader` has `isClosed` method in the same situation.

Reply:
@SharplEr well, ok, the method name can stay unchanged, but why not use EmptyClosableReader then? It provides the same functionality.

Reply:
@proshin-roman I can't use here `EmptyClosableReader` because it extends `Reader`, but I need `Writer`

Reply:
@SharplEr ok, no more questions :)